helpers/buildroot/build-gnu-toolchain.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eux -o pipefail
+shopt -qs failglob
+
+for opt in "$@"; do
+   optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+   case "${opt}" in
+      --arch=*) ARCH="${optarg}" ;;
+      --kernel-version=*) KVER="${optarg}" ;;
+   esac
+done
+
+ARCH="${ARCH:?}"
+KVER="${KVER:?}"
+
+cd "${HOME}/buildroot"
+make O="output/${ARCH}-gnu defconfig" BR2_DEFCONFIG="configs/sdk_${ARCH}_gnu_defconfig"
+make O="output/${ARCH}-gnu toolchain"
```

Comment:
I messed up the quoting here - will push a fix once I've tested the rest of the build.